index.js

```diff
@@ -55,284 +48,117 @@ process
-      parsedInsultTriggers.forEach(trigger => { // Loop ever each insult and check if they match the message
-        possible_LCMs.forEach(unleeted_LCM => { // Loop over every possible unleeted message to match with insult
-          if (new RegExp(trigger, "g").test(unleeted_LCM)) {
-            commonElements.push(true)
-          }
-        });
-      })
-      if (commonElements.length > 0) {
-        const insult = new MessageEmbed()
-          .setColor('#04d384')
-          .setTitle(`${insulttitle}`)
-          .setAuthor(`${insultauthor}`, 'https://spbot.ml/siround.png')
-          .setDescription(`${insultdescription}`)
-        message.channel.send({ embeds: [insult] }).catch(console.err)
-      }
-    }
-  } 
+  // Verify permissions of user who sent message before continuing.
+  if (message.author.bot || message.channel.type === 'DM' || !message.channel.permissionsFor(client.user).has(botPerms)) return;
 
+  let lang = "en";
+  const server_language = db.get(`lang_${message.guild.id}`);
+  if (typeof server_language === 'string') lang = server_language;
 
+  let LCM = message.content.toLowerCase(); //Lower case message text
 
+  // Mention bot will activate alert message without triggers
+  if (message.mentions.users.first() === client.user)
+    return require('./split/bot-mentioned')(message, lang);
 
+  if (db.get(`mute_${message.author.id}`) == null) //Check to see if you muted the bot (User side only)
+    require('./split/every-unmuted-message')(message, lang);
 
-  //----------BEGIN COMMANDS AREA----------
-  //Ping command
-  const loading = client.emojis.cache.get("838616104687108117");
-  if (LCM === '(ping' || LCM === ')ping' || LCM === '\\ping' || LCM === '~ping' || LCM === '|ping' || LCM === '!ping' || LCM === '?ping' || LCM === '.ping' || message.content === '$ping' || LCM === '%ping' || LCM === '-ping' || LCM === '--ping' || LCM === '=ping' || LCM === '+ping' || LCM === '_ping' || LCM === '/ping' || LCM === '&ping' || LCM === '--ping' || LCM === "`ping" || LCM === 'sp!ping') {
-    message.channel.send(`${loading} Pinging...`).then((msg) => {
-      msg.edit("\u200B");
-      const ping = new MessageEmbed()
-        .setColor('#04d384')
-        .setTitle('Here is my ping young one')
-        .setDescription(`Roundtrip latency is ${Math.floor(msg.createdTimestamp - message.createdTimestamp)}ms \nAPI Latency is ${Math.round(client.ws.ping)}ms`)
-      msg.edit({ embeds: [ping] });
-    });
-    return;
-  }
+  // #region Commands
+  // Ping command
+  
+  if (LCM === '(ping' || LCM === ')ping' || LCM === '\\ping' || LCM === '~ping' || LCM === '|ping' || LCM === '!ping' || LCM === '?ping' || LCM === '.ping' || LCM === '$ping' || LCM === '%ping' || LCM === '-ping' || LCM === '--ping' || LCM === '=ping' || LCM === '+ping' || LCM === '_ping' || LCM === '/ping' || LCM === '&ping' || LCM === '--ping' || LCM === "`ping" || LCM === 'sp!ping')
+    return require('./commands/ping')(message, lang);
```

Comment:
stop reading the files upon every call upon the command files
use a basic command handler